vonk/facade/enablesearch_2.rst

```diff
@@ -86,6 +86,13 @@ The gist of the implementation is to switch the querying based on the ResourceTy
          return new SearchResult(patientResources, query.GetPageSize(), count);
      }
 
+* Note that the information model is based on STU3 by default. When using FHIR version R4 or higher you will need to override the EntryInformationModel. The implementation of this looks like::
```

Comment:
I'm a little confused about the context of this, what happens if you don't override `EntryInformationModel` and what happens if you do?

Reply:
It is indeed a little confusing. If you don't override it, and R4 call will lead to a NotImplementedException. This is done because older plugins always implement R3 only (and do not filter on version). Those plugins will thus continue to function. For R4, you need to suppress the NotImplementedException by overriding the method. Please note that this could mean you adding a version filter to the query. But if you only do R4 and you don't want to use a filter, you can use the override described here

Reply:
That makes a lot of sense! I think this should be included in the docs.

Reply:
Yes! Sebastiaan made a card for that Thursday. He had the same problem